snow/engine/avalanche/bootstrap/bootstrapper.go

```diff
@@ -52,41 +54,41 @@ func New(
 			OnFinished: onFinished,
 		},
 	}
-
-	if err := b.metrics.Initialize("bs", config.Ctx.AvalancheRegisterer); err != nil {
-		return nil, err
-	}
-
-	config.Config.Bootstrapable = b
-	b.Bootstrapper = common.NewCommonBootstrapper(config.Config)
-	return b, nil
+	return b, b.metrics.Initialize("bs", config.Ctx.AvalancheRegisterer)
 }
 
 // Note: To align with the Snowman invariant, it should be guaranteed the VM is
 // not used until after the bootstrapper has been Started.
 type bootstrapper struct {
 	Config
+	common.Halter
 
 	// list of NoOpsHandler for messages dropped by bootstrapper
 	common.StateSummaryFrontierHandler
 	common.AcceptedStateSummaryHandler
+	common.AcceptedFrontierHandler
```

Comment:
q: this is a no-op since post linearization we'll never ever ask a DAG frontier to any peer right? We'll go for the hard coded version of with whatever we have already downloaded and store locally